api/package.json

```diff
@@ -28,6 +28,7 @@
     "mongodb": "4.17.1",
     "nanoid": "3",
     "no-profanity": "1.5.1",
+    "node-fetch": "2.7.0",
```

Comment:
I am not going to block on this, but why use `node-fetch` at all? Then, why use `v2` when `v3` has been out for a while?

Reply:
> node-fetch from v3 is an ESM-only module - you are not able to import it with require().
If you cannot switch to ESM, please use v2 which remains compatible with CommonJS. Critical bug fixes will continue to be published for v2.

We should be able to use v3

Reply:
> but why use node-fetch at all?

Good point, we're on node 20 and can drop this. Thanks for the reminder!

> We should be able to use v3

It's moot now that node supports fetch natively, but we couldn't use v3 since it's a module and we need to be able to `require` it.